src/Config/Converters/NestedMutationOptionsConverter.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Azure.DataApiBuilder.Config.ObjectModel;
+
+namespace Azure.DataApiBuilder.Config.Converters
+{
+    /// <summary>
+    /// Converter for the nested mutation options.
+    /// </summary>
+    internal class NestedMutationOptionsConverter : JsonConverter<NestedMutationOptions>
+    {
+
+        private readonly NestedCreateOptionsConverter _nestedCreateOptionsConverter;
+
+        public NestedMutationOptionsConverter(JsonSerializerOptions options)
+        {
+            _nestedCreateOptionsConverter = options.GetConverter(typeof(NestedCreateOptions)) as NestedCreateOptionsConverter ??
+                                            throw new JsonException("Failed to get nested create options converter");
+        }
+
+        /// <inheritdoc/>
+        public override NestedMutationOptions? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            if (reader.TokenType == JsonTokenType.Null)
```

Comment:
To simplify usage, is it possible to have:

`"nested-mutations" : true` 
to imply any supported nested mutations are enabled. Could be a separate PR.

Reply:
This would involve a slightly broader discussion and also requires a new option. 

The current option `--graphql.nested-create.enabled` at the granularity level of `create` operation. To enable/disable something that is at the level of all mutation operations, we will need a new option. 

Ideally, should be taken up in a new issue after discussion.

Reply:
I've created this issue --->  [#2089 ](https://github.com/Azure/data-api-builder/issues/2089)